utils/config.ts

```diff
@@ -12,22 +12,52 @@ import { pipe } from "fp-ts/lib/function";
 
 import { readableReport } from "@pagopa/ts-commons/lib/reporters";
 import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { withDefault } from "@pagopa/ts-commons/lib/types";
 
 // global app configuration
-export type IConfig = t.TypeOf<typeof IConfig>;
-// eslint-disable-next-line @typescript-eslint/ban-types
-export const IConfig = t.interface({
-  AzureWebJobsStorage: NonEmptyString,
-
+export type IDecodableConfig = t.TypeOf<typeof IDecodableConfig>;
+export const IDecodableConfig = t.interface({
+  COSMOSDB_COLLECTION: NonEmptyString,
```

Comment:
Let's be more specific about this variable. Maybe `COSMOSDB_SERVICE_COLLECTION`? (same with lease variable)

Reply:
Fix with b22b0f4